src/fs/filesystem.ts

```diff
@@ -98,27 +111,39 @@ export class FileSystem {
   static async empty(opts: FileSystemOptions = {}): Promise<FileSystem> {
     const { keyName = 'filesystem-root', rootDid = '' } = opts
 
-    const root = await BareTree.empty()
-    const publicTree = await PublicTree.empty(null)
+    const publicTree = await PublicTree.empty()
     const prettyTree = await BareTree.empty()
-    const pinsTree = await BareTree.empty()
 
+    const mmpt = MMPT.create()
     const key = await keystore.getKeyByName(keyName)
-    const privateTree = await PrivateTree.empty(key)
+    const privateTree = await PrivateTree.create(mmpt, key, null)
+
+    const root = await BareTree.empty()
 
-    await root.addChild('public', publicTree)
-    await root.addChild('pretty', prettyTree)
-    await root.addChild('private', privateTree)
-    await root.addChild('pins', pinsTree)
+    // await root.addChild('public', publicTree)
+    // await root.addChild('pretty', prettyTree)
+    // await root.addChild('private', privateTree)
```

Comment:
Still needed?

Reply:
nope, thanks! forgot to cleanup this file :sweat_smile: